src/gui/optionsdlg.cpp

```diff
@@ -165,83 +165,88 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
 
     // Connect signals / slots
-    connect(m_ui->comboProxyType, SIGNAL(currentIndexChanged(int)), this, SLOT(enableProxy(int)));
-    connect(m_ui->checkRandomPort, SIGNAL(toggled(bool)), m_ui->spinPort, SLOT(setDisabled(bool)));
+    // shortcuts for frequently used signals and slots
```

Comment:
No, I don't. I didn't understand your reason. Your comment don't describe it.
In case of preventing type casting it's a good idea, but I still don't like names you chose. I would use full type name as prefix instead of your weird "qcb" and so on (e.g. QCheckBoxToggled).